config/software/datadog-agent-integrations.rb

```diff
@@ -0,0 +1,85 @@
+require './lib/ostools.rb'
+
+name 'datadog-agent-integrations'
+
+dependency 'pip'
+dependency 'datadog-agent'
+
+relative_path 'integrations-core'
+
+# The only integrations that will be packaged with the agent
+# are the ones that are officiallly supported.
+local_integrations_core_repo = ENV['LOCAL_INTEGRATIONS_CORE_REPO']
+if local_integrations_core_repo.nil? || local_integrations_core_repo.empty?
+  source git: 'https://github.com/DataDog/integrations-core.git'
+else
+  # For local development
+  source path: ENV['LOCAL_INTEGRATIONS_CORE_REPO']
+end
+
+integrations_core_branch = ENV['INTEGRATION_CORE_BRANCH']
+if integrations_core_branch.nil? || integrations_core_branch.empty?
+  default_version 'master'
+else
+  default_version integrations_core_branch
+end
+
+build do
+  # Agent code
+  mkdir "#{install_dir}/agent/checks.d"
+
+  # Grab all the checks
+  checks = Dir.glob("#{integrations_dir}/*/")
+  # Only use the parts of the filename we need
+  check.slice! "#{integrations_dir}/"
```

Comment:
Not the right place, needs to be after `checks.each`? Or can you apply `slice` to each `checks` globally?